src/data-table/index.js

```diff
@@ -189,8 +195,13 @@ class DataTable extends Component {
       ? this.getFilteredData()
       : data
 
-    // sorting, todo: remove lodash
-    const sortedData = orderBy(filteredData, [sortColumn], sortDirection.slice(0, -6))
+    const columns = this.pickedColumns()
+
+    const sortedData = sortType ? filteredData.sort(
```

Comment:
looks like `sortType`'s default is not being applied here as the dataTable just refuse to do sort without a `sortType` even if `sortable` is `true`

Reply:
I think i know why this is happening, let me quick make a fix PR